### DISCUSSION TITLE
Same integration for FOSSIBOT power stations?

### DISCUSSION THREAD

**Discussion by iamslan** [tolwi/hassio-ecoflow-cloud#376] [Q&A]
Hello,
Not sure this is the place, but I'd love to get the equivalent quality integration for Fossibot power stations.
I don't know a thing about hassio integrations, but I did a full reverse engineering of fossibot API if someone wants to implement this to HA.
This is me tossing a bottle into the sea!

**Comment by loomi42** [comment]
Hi iamslan. I now own a Fossibot 2400, but instead of HA integration I want to integrate using shell scripts (or python) on Linux. Would be very happy to get the API documentation. In return I will for sure share my results, i.e. shell scripts.

**Reply by iamslan** [reply]
Check my GitHub you have a ready HA integration and a nodejs implementation.
